fetcher.go

```diff
@@ -219,12 +220,13 @@ func (p *pool) fetchResource(ctx context.Context, from string, resource string,
 					HTTPProtocol:       proto,
 					TTFBMS:             int(ttfbMs),
 					// my address
-					Range:          "",
-					Referrer:       respReq.Referer(),
-					UserAgent:      respReq.UserAgent(),
-					NodeId:         saturnNodeId,
-					NodeIpAddress:  from,
-					IfNetworkError: networkError,
+					Range:             "",
+					Referrer:          respReq.Referer(),
+					UserAgent:         respReq.UserAgent(),
+					NodeId:            saturnNodeId,
+					NodeIpAddress:     from,
+					IfNetworkError:    networkError,
+					VerificationError: verificationError,
```

Comment:
how does this get set?  / how is this ever not ""

Reply:
See line 360 below ?